src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/FiltersForm.test.tsx

```diff
@@ -222,6 +222,7 @@ describe('FiltersForm', () => {
   const testWizardStepProps: InjectedWizardProps = {
     currentStep: 1,
     isActive: true,
+    isEnabled: true,
```

Comment:
Is there any test covering the scenario that isEnabled is false?

Reply:
Test cases around this have been added to `Wizard.test.tsx` as this. As it's only consumed by `WizardStepHeading` to toggle some classes, there isn't really any need to test this further.

Reply:
Cool.